fontbe/src/error.rs

```diff
@@ -12,7 +12,7 @@ use write_fonts::tables::{glyf::BadKurbo, gvar::GvarInputError, variations::IupE
 pub enum Error {
     #[error("IO failure")]
     IoError(#[from] io::Error),
-    #[error("Fea compilation failure")]
+    #[error("FEA compilation failure {0}")]
```

Comment:
+1 to Display, -1 to FEA in all caps. Alternatives: ".fea compilation failure" or "Feature file compilation failure"